hmy/staking.go

```diff
@@ -133,6 +133,14 @@ func (hmy *Harmony) IsPreStakingEpoch(epoch *big.Int) bool {
 	return hmy.BlockChain.Config().IsPreStaking(epoch)
 }
 
+// IsCommitteeSelectionBlock checks if the given block is the committee selection block
+func (hmy *Harmony) IsCommitteeSelectionBlock(blk *types.Block) bool {
+	isBeaconChain := blk.ShardID() == shard.BeaconChainShardID
+	isNewEpoch := len(blk.Header().ShardState()) > 0
```

Comment:
is there any util function to check isnewepoch?  this is a very common operation, maybe we can add a function is not alrready.

Reply:
Sure, the last commit of this PR refactored this part.